webrtc.html

```diff
@@ -720,7 +720,7 @@
                   </li>
 
                   <li>
-                    <p>Fire a simple event named <code><a href=
+                    <p>Queue a task to fire a simple event named <code><a href=
```

Comment:
@adam-be the difference would only be observable in a JS debugger, because a script can only learn of a rejection through functions passed to `.then` or `.catch`, which are guaranteed to always execute on a microtask.